apps/standalone/src/app/context/AuthContext.ts

```diff
@@ -125,20 +129,21 @@ export const useAuthContext = () => {
                   errorMessage = text;
                 }
               }
+              console.log('%c error message for failed userinfo', 'color: red; font-size:18px', errorMessage);
             } catch (parseErr) {
               // If parsing fails, use default error message
             }
             setError(errorMessage);
-            // Don't redirect if we're already on the login page
-            if (window.location.pathname !== '/login') {
+            // Don't redirect if we're already on the login or login callback pages
+            if (!['/login', '/callback'].includes(window.location.pathname)) {
```

Comment:
@rawagner this extra redirect was causing the error coming from the UI proxy / backend request to be lost.